[2023-10-22T10:24:44.637+0000] {taskinstance.py:1157} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: etl_day18.fetch_data_from_api manual__2023-10-22T10:24:40.775474+00:00 [queued]>
[2023-10-22T10:24:44.657+0000] {taskinstance.py:1157} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: etl_day18.fetch_data_from_api manual__2023-10-22T10:24:40.775474+00:00 [queued]>
[2023-10-22T10:24:44.658+0000] {taskinstance.py:1359} INFO - Starting attempt 1 of 1
[2023-10-22T10:24:44.687+0000] {taskinstance.py:1380} INFO - Executing <Task(PythonOperator): fetch_data_from_api> on 2023-10-22 10:24:40.775474+00:00
[2023-10-22T10:24:44.697+0000] {standard_task_runner.py:57} INFO - Started process 61 to run task
[2023-10-22T10:24:44.709+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'etl_day18', 'fetch_data_from_api', 'manual__2023-10-22T10:24:40.775474+00:00', '--job-id', '260', '--raw', '--subdir', 'DAGS_FOLDER/ETL_day18.py', '--cfg-path', '/tmp/tmpfvuhrnig']
[2023-10-22T10:24:44.730+0000] {standard_task_runner.py:85} INFO - Job 260: Subtask fetch_data_from_api
[2023-10-22T10:24:44.857+0000] {task_command.py:415} INFO - Running <TaskInstance: etl_day18.fetch_data_from_api manual__2023-10-22T10:24:40.775474+00:00 [running]> on host 90cf752d844d
[2023-10-22T10:24:45.512+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='kel11' AIRFLOW_CTX_DAG_ID='etl_day18' AIRFLOW_CTX_TASK_ID='fetch_data_from_api' AIRFLOW_CTX_EXECUTION_DATE='2023-10-22T10:24:40.775474+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2023-10-22T10:24:40.775474+00:00'
[2023-10-22T10:24:46.624+0000] {python.py:194} INFO - Done. Returned value was: None
[2023-10-22T10:24:46.634+0000] {taskinstance.py:1398} INFO - Marking task as SUCCESS. dag_id=etl_day18, task_id=fetch_data_from_api, execution_date=20231022T102440, start_date=20231022T102444, end_date=20231022T102446
[2023-10-22T10:24:46.687+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2023-10-22T10:24:46.924+0000] {taskinstance.py:2776} INFO - 1 downstream tasks scheduled from follow-on schedule check
